src/commonMain/kotlin/io/modelcontextprotocol/kotlin/sdk/client/SSEClientTransport.kt

```diff
@@ -39,7 +39,7 @@ public class SseClientTransport(
     private var job: Job? = null
 
     private val baseUrl by lazy {
-        session.call.request.url.toString().removeSuffix("/")
+        session.call.request.url.toString().removeSuffix("/sse")
```

Comment:
I would prefer to keep the prefix to have an explicit URL, but use the correct URL concatenation method to avoid duplicate // in path segments. There are several methods for `pathSegment` in Ktor you can use or build your own.